sim/paladin/sor.go

```diff
@@ -103,7 +103,7 @@ func (paladin *Paladin) registerSealOfRighteousness() {
 			ActionID:    core.ActionID{SpellID: rank.proc.spellID},
 			SpellSchool: core.SpellSchoolHoly,
 			DefenseType: core.DefenseTypeMelee,
-			ProcMask:    core.ProcMaskMeleeMHSpecial, //changed to ProcMaskMeleeMHSpecial, to fix procs from weapons/oils which do proc from SoR, Idk side effects, does WS proc from SoR?
+			ProcMask:    core.ProcMaskMeleeMHSpecial | core.ProcMaskSupressExtraAttack,//changed to ProcMaskMeleeMHSpecial, to allow procs from weapons/oils which do proc from SoR, but Wild Strikes does not proc
```

Comment:
For completeness could you add this to the Windfury code below as well? Even though it doesn't apply to paladins obviously who knows if something else for another class might end up in the same situation

Reply:
couldn't hurt